.npmrc

```diff
@@ -1 +1,4 @@
 cache = ~/.atom/.npm
+runtime = electron
+target = 0.34.5
```

Comment:
I don't like that we have to put this here and in package.json. Anyone have ideas about how we could not do that?